pages/credits.vue

```diff
@@ -46,10 +46,10 @@
 				<Column header="Amount" field="amount" class="md:w-44 2xl:w-64">
 					<template #body="slotProps">
 						<Tag v-if="slotProps.data.type === 'addition'" class="flex items-center !text-sm" severity="success">
-							<nuxt-icon class="mr-2 mt-0.5" name="coin"/>+{{ (slotProps.data.amount || 0).toLocaleString('en-US') }}
+							<NuxtIcon class="mr-2" name="coin" aria-hidden="true"/>+{{ (slotProps.data.amount || 0).toLocaleString('en-US') }}
 						</Tag>
 						<Tag v-else class="flex items-center !text-sm" severity="danger">
-							<nuxt-icon class="mr-2 mt-0.5" name="coin"/>-{{ (slotProps.data.amount || 0).toLocaleString('en-US') }}
+							<NuxtIcon class="mr-2" name="coin" aria-hidden="true"/>-{{ (slotProps.data.amount || 0).toLocaleString('en-US') }}
```

Comment:
Not sure about the margin change here, let's recheck after the nuxt icon switch.